packages/services/src/balances/runes-balances.service.ts

```diff
@@ -130,7 +133,15 @@ export class RunesBalancesService {
       )
     )
       .filter(result => result.status === 'fulfilled')
-      .map(b => b.value);
+      .map(b => b.value)
+      .filter(ft => {
```

Comment:
Maybe this filter can be extracted and shared between `runes` + `sip-10`? It seems to be doing the same thing.

I think it could also be refactored to:
```
.filter({asset} => {

          request.filters?.assetVisibility?.[serializeAssetId(getAssetId(asset))] || true
      });

```